src/openforms/contrib/zgw/clients/zaken.py

```diff
@@ -130,3 +130,22 @@ def create_rol(
         response.raise_for_status()
 
         return response.json()
+
+    def create_zaakobject(
+        self, zaak: dict, object: str, objecttype_version: str
+    ) -> dict | None:
```

Comment:
```suggestion
    ) -> dict:
```
I don't see a code path that can return `None`